frontend/src/features/admin-form/responses/AdminSubmissionsService.ts

```diff
@@ -24,3 +33,22 @@ export const countFormSubmissions = async ({
   }
   return ApiService.get(queryUrl).then(({ data }) => data)
 }
+
+/**
+ * Get of submissions metadata for a given form
+ * @param formId fomID to retrieve metadata on
+ * @returns The metadata of the form
+ */
+export const getFormSubmissionsMetadata = async (
+  formId: string,
+): Promise<StorageModeSubmissionMetadataList> => {
+  const queryUrl = `${ADMIN_FORM_ENDPOINT}/${formId}/submissions/metadata?page=1` // To be generalized in later PRs
```

Comment:
we should generalize this now, it may be missed in later PRs. there is an existing method that should be a copy-paste (and changing axios to ApiService) in `src/public/services/AdminSubmissionsService.ts#getSubmissionsMetadataByPage`

Reply:
Sure, i think that will be better too. Will generalize this function and give it a default value = 1 in the params since there is no pagination yet